src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -527,7 +527,7 @@ private void mapCustomFields(ScanRequest request, ScanResults.XIssue issue, Issu
                                 if (issue.getLink() != null && !issue.getLink().isEmpty()) {
                                     recommendation.append("Checkmarx Link: ").append(issue.getLink()).append(HTMLHelper.CRLF);
                                 }
-                                if (!ScanUtils.empty(issue.getCwe())) {
+                                if (!ScanUtils.anyEmpty(flowProperties.getMitreUrl(), issue.getCwe())) {
```

Comment:
what do we do if there is no Mitre? do we just ignore it?

Reply:
yes. it should be empty